### PR TITLE
Issues/258 add python to hera

### PR DESCRIPTION
**Pull request by jvivian**
Includes a bump to 1.1 for bug fixes that were in 1.0.1

### CODE REVIEW

File: hera/Dockerfile [hunk 1]
```diff
@@ -8,13 +8,14 @@ RUN apt-get update  && apt-get install -y \
...
-		wget
+		wget \
+		python
 
 # install hera from source
 WORKDIR /
 RUN git clone https://github.com/bioturing/hera.git
 WORKDIR /hera
-RUN git checkout 53dd2a1927d86d2299f16cf265d3f0e259c0e3df
+RUN git checkout 4ed3c2cf0eabcb5812c4b1520c72192b7aaf80c7
```
Comment (fnothaft) [inline hera/Dockerfile]: Do they have a 1.1 tag? If so, I find checking out the tag to be preferable to specifying the commit hash.

**Review by fnothaft — APPROVED** [review]
LGTM, let me know if you want to/can resolve my comment wrt them having the release tag or not.

File: hera/Dockerfile (hunk 1 above)
Comment (jvivian) [reply hera/Dockerfile]: Good call thanks

**Comment by jvivian** [comment]
@fnothaft — Resolved

**Review by fnothaft — APPROVED** [review]

**Comment by fnothaft** [comment]
Merged! Thanks @jvivian!

**Pull request merged**